backend/app/main.py

```diff
@@ -72,6 +72,22 @@ def get_session():
         yield session
 
 
+def update_timestamp(
+    session: Session,
+    document_id: str,
+    updated_at: str,
+    execute: bool = True,
+):
+    update_stmt = (
+        update(Document)
+        .where(Document.document_id == document_id)
+        .values(updated_at=updated_at)
+    )
+    if execute:
+        session.execute(update_stmt)
+        session.commit()
```

Comment:
And let the parent caller commit.

```suggestion
    session.execute(update_stmt)
```

Reply:
Ahh thank you -- apologies for the misfire on my end :)